assopy/admin.py

```diff
@@ -495,7 +498,7 @@ def clean_coupon(self):
             'user': user,
             'form': form,
         }
-        return render_to_response('admin/auth/user/new_order.html', ctx, context_instance=template.RequestContext(request))
+        return TemplateResponse('admin/auth/user/new_order.html', ctx)
```

Comment:
one of the API changes is that TemplateResponse takes request as first argument, I think it needs to be passed here as well. (and all other TemplateResponse uses)

Reply:
Thanks for catching this, got it mixed up with `SimpleTemplateResponse`. I guess admin views are not covered in tests that's why this didn't raise any flags in CI. Will address shortly.

Reply:
maybe we should also add basic HTTP 200 tests/stubs for those views, while there's no request there, so make sure they all fail, so that they work after this change? :)

Reply:
I'm up for doing that - the only concern being the PR getting too large (the diff is at ~600 lines right now) and I won't be able to do it until the weekend/early next week.

Reply:
take your time, don't need to rush this upgrade. :)
as for lines – yeah, unfortunately. 
we could try merging that separately before this PR (the testsuite should still be compatible with both "versions"), but at the same time, I don't mind one big PR either – it's more important to include tests for the code that's being changed.